apollo-federation/src/link/federation_spec_definition.rs

```diff
@@ -495,24 +496,7 @@ impl FederationSpecDefinition {
             })
     }
 
-    // The directive is named `@fromContex`. This is confusing for clippy, as
```

Comment:
Conventions:
- This is just a small nit, though spec files seem to have a pattern/order of definition getter, application creator, and argument extractor.